packages/server/src/modules/i18n/index.js

```diff
@@ -0,0 +1,22 @@
+import i18n from 'i18n';
+
+import Feature from '../connector';
+import settings from '../../../../../settings';
+
+const config = {
+  directory: process.cwd() + '/src/modules/i18n/locales',
```

Comment:
I have told this earlier - you should have per-module locales, not global locales for the whole backend. Please check how each module has own locales in frontend and do the same for backend